src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CancellationTokenParametersMustComeLastTests.cs

```diff
@@ -418,5 +418,96 @@ public class C
     }
 }");
         }
+
+        [Theory, WorkItem(2851, "https://github.com/dotnet/roslyn-analyzers/issues/2851")]
+        // General analyzer option
+        [InlineData("public", "dotnet_code_quality.api_surface = public")]
```

Comment:
Can we add a scenarios to the new tests with empty editorconfig and verify it flags all accessibility APIs?